python/pyspark/sql/dataframe.py

```diff
@@ -519,7 +520,7 @@ def sort(self, *cols, **kwargs):
             jcols = [jc if asc else jc.desc()
                      for asc, jc in zip(ascending, jcols)]
         else:
-            raise TypeError("ascending can only be bool or list, but got %s" % type(ascending))
+            raise TypeError("ascending can only be boolean or list, but got %s" % type(ascending))
```

Comment:
I think bool is the name of the function, but the data type is still boolean?

see

http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.sort.html#pandas.DataFrame.sort

https://docs.python.org/2/library/stdtypes.html#boolean-values

https://docs.python.org/2/c-api/bool.html